cmds/graph.c

```diff
@@ -813,7 +813,14 @@ static bool print_task_node(struct uftrace_task *task,
 
 	print_task_field(task);
 	pr_indent(indent_mask, indent, true);
-	pr_out("%s\n", name);
+	if (parent && parent->pid == task->pid) {
+		/* print thread name in green color */
+		pr_green("%s\n", name);
```

Comment:
I'm not sure about this..  We already use different indent and now you added '*' too.

Reply:
I actually prefer using color rather than indentation because it required more explanation for new users. Even I didn’t understand why their indentation is different and I thought it looked like a bug.

Reply:
But color has no meaning what it is, so you still need an explanation.  From the indentation we can guess whether it's a closer descendant (thread) or not (process) IMO.

Reply:
Also you didn't fix(?) the indentation.

Reply:
I think we can use both indentation and green color. I just removed asterisk for process, but didn't change for indentation and color.